File: app/src/main/java/com/dev/fitface/models/LoginResponse.java
package com.dev.fitface.models;

import com.google.gson.annotations.SerializedName;

public class LoginResponse extends BaseResponse {
    @SerializedName("data")
    UserInfo data;

    public LoginResponse(String message, int status) {
        super(message, status);
    }

    public UserInfo getData() {
        return data;
    }

    public void setData(UserInfo data) {
        this.data = data;
    }
}

File: app/src/main/java/com/dev/fitface/models/response/LoginResponse.kt
package com.dev.fitface.models.response

import com.dev.fitface.models.User
import com.google.gson.annotations.SerializedName

data class LoginResponse(
        @SerializedName("status")
        var status: Int?,
        @SerializedName("message")
        var message: String?,
        @SerializedName("data")
        var data: User?
)